### ISSUE TITLE
Missing `rpushBuffer`

### ISSUE THREAD

**Issue by kevinhu** [luin/ioredis#1723]
I recently upgraded from v4 to v5, and I have a section of code that uses `Redis.rpushBuffer`. It appears that this function is still present but not annotated in the built-in types.

Running the following snippet will produce the output `[Function (anonymous)]`, despite the code raising a type error when the `// @ts-ignore` is removed:

```typescript
import { Redis } from "ioredis";

const r = new Redis(
  {
    host: "localhost",
    port: 6379,
    password: "thisisapassword",
  },
);

// @ts-ignore
console.debug(r.rpushBuffer);
```

**Comment by luin** [comment]
Yeah v5 only provides declarations for methods that we consider useful. It is worth noting that you don't need the `Buffer` variant to send buffers, so you should go with `rpush(Buffer.from(...))`
